bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/util/RulePredicates.java

```diff
@@ -0,0 +1,172 @@
+/**
+ * Copyright (c) 2017 by Deutsche Telekom AG and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.automation.core.util;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+
+import com.google.common.base.Predicate;
+
+import org.eclipse.smarthome.automation.Rule;
+import org.eclipse.smarthome.automation.core.internal.RuleEngine;
+
+/**
+ * Add support for scope prefix and provide default predicates for scope and tags.
+ *
+ * @author Victor Toni - initial contribution
+ * @author Victor Toni - provide default predicates for scope and tags
+ *
+ */
+public class RulePredicates {
+
+    /**
+     * Constant defining separator between module UID and output name.
+     */
+    public static final char SCOPE_DELIMITER = ':';
```

Comment:
I thought we said that we do not introduce any notion of a "scope" anywhere - instead, I would simply keep calling it prefix or namespace.